src/sagemaker/jumpstart/model.py

```diff
@@ -786,6 +791,70 @@ def register_deploy_wrapper(*args, **kwargs):
 
         return model_package
 
+    def display_benchmark_metrics(self):
+        """Display Benchmark Metrics for deployment configs."""
+        config_names = []
+        latencies = []
+        through_puts = []
+        instance_types = []
+        instance_rates = []
+        for deployment_config in self.list_deployment_configs():
```

Comment:
If there is a 1-1 mapping between deployment_config to benchmark metric, it needs to ideally be modeled as an object. 

https://github.com/huggingface/optimum-benchmark/blob/main/optimum_benchmark/benchmarks/inference/benchmark.py

https://github.com/huggingface/optimum-benchmark/blob/c561159787476c962f9c9ab085d744520ad359f8/optimum_benchmark/benchmarks/base.py#L12